apollo-client/src/main/java/com/ctrip/framework/apollo/util/http/DefaultHttpClient.java

```diff
@@ -19,14 +19,14 @@
 /**
  * @author Jason Song(song_s@ctrip.com)
  */
-public class HttpUtil {
```

Comment:
I think we may still keep the class `HttpUtil` and delegate its implementation to the `HttpClient` because some application might already reference it. I did a [quick search](https://github.com/qw7575/spring-boot-examples/blob/9dd0b838262ea86c018684cb6218c1540039e64a/spring-boot-jsoup/src/main/java/com/qw/boot/examples/jsoup/service/LogicServiceImpl.java) in github, and found the possibility is true.

To avoid future misuse, we could mark it as `@Deprecated`.